src/middleware/auth.ts

```
@@ -1,7 +1,35 @@
 import type { FastifyRequest, FastifyReply } from "fastify";
+import type { FastifyRequest, FastifyReply } from "fastify";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FastifyRequest'. Consider to delete it or rename it.

'FastifyRequest' is defined here:


(lint/suspicious/noRedeclare)